### PR TITLE
[MSVCRT_WINETEST] Mute _CRT_NON_CONFORMING_SWPRINTFS

### PR DESCRIPTION
**Pull request by learn-more**

### CODE REVIEW

File: modules/rostests/winetests/msvcrt/precomp.h [hunk 1]
```diff
@@ -6,7 +6,9 @@
 #define _INC_WINDOWS
 #define COM_NO_WINDOWS_H
 
+#if !defined(_CRT_NON_CONFORMING_SWPRINTFS)
```
Comment (HBelusca) [inline modules/rostests/winetests/msvcrt/precomp.h:9]: ifndef should be shorter ^^

Comment (JoachimHenze) [reply modules/rostests/winetests/msvcrt/precomp.h:9]: One day he is gonna kill you :D

Comment (learn-more) [reply modules/rostests/winetests/msvcrt/precomp.h:9]: just block his github access

**Review by JoachimHenze — APPROVED** [review]
[build-msvc (14.1, i386, Debug)](https://github.com/reactos/reactos/runs/5876890077?check_suite_focus=true#logs)
does not longer warn.
(I also think ifndef is better, but I won't block due to that)

**Pull request merged**